.taskmaster/error-handler.py

```
@@ -0,0 +1,350 @@
+    
+    def send_escalation_alert(self, escalation):
+        """Send escalation alert to master control interface"""
+        alert_file = os.path.join(PROJECT_ROOT, '.taskmaster', 'alerts.json')
+        
+        # Load existing alerts
+        alerts = []
+        if os.path.exists(alert_file):
+            try:
+                with open(alert_file, 'r') as f:
+                    alerts = json.load(f)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)